README.md

```diff
@@ -5,7 +5,7 @@ Feel like npm is drunk or maybe you are and want to verify what gets published v
 ```sh
 ➝ irish-pub
 
-npm publish will include the following files:
+Publishing irish-pub@1.0.0 as thlorenz
```

Comment:
Change to:

*npm publish will publish irish-pub@1.0.0 as thlorenz including the following files:*

in order to make clear that nothing is actually getting published.

Also change the code accordingly to get that message.